template-manager-cmtemplate/src/main/java/com/sequenceiq/cloudbreak/cmtemplate/CMRepositoryVersionUtil.java

```diff
@@ -32,6 +32,8 @@
 
     public static final Versioned CLOUDERAMANAGER_VERSION_7_2_6 = () -> "7.2.6";
 
+    public static final Versioned CLOUDERAMANAGER_VERSION_7_2_7 = () -> "7.2.7";
```

Comment:
mmm so we are switching to Runtime and CM versions being different starting from 7.2.7. I

The question now becomes if you should check for the CM version or Cloudera Runtime (CDH) version. If it is Cloudera Runtime version, then we should update the check on runtime (instead of CM).

If this should be a CM check, then there is no CM 7.2.7. After CM 7.2.6, it becomes CM 7.3.0. You should follow up with [CRB-1455](https://jira.cloudera.com/browse/CRB-1455) to understand more about how to check runtime version vs. CM version.

Reply:
This is not a problem
1) I get the runtime version from blueprint as input of the check, not CM version. See https://github.com/hortonworks/cloudbreak/blob/master/template-manager-cmtemplate/src/main/java/com/sequenceiq/cloudbreak/cmtemplate/configproviders/hbase/HbaseCloudStorageServiceConfigProvider.java#L66
2) Although the constant used is defined in the CM utility, we are comparing the stack runtime version against a constant. So the check is still valid even when CM version is different from runtime version. What matters is the meaning of the input variable. see https://github.com/hortonworks/cloudbreak/blob/master/template-manager-cmtemplate/src/main/java/com/sequenceiq/cloudbreak/cmtemplate/configproviders/hbase/HbaseCloudStorageServiceConfigProvider.java#L57

Reply:
Ah that makes sense. Thanks for the information @LinaAtAustin 

since we are checking for the runtime version and not the CM version, lets change the constant name to "CLOUDERA_RUNTIE_VERSION_7_2_7", so its not confusing for future maintainers.

Reply:
@kmanamcheri  please open a backburner ticket for this.

Reply:
ok done https://jira.cloudera.com/browse/CB-10353